fuse_variables/test/test_fixed_size_variable.cpp

```diff
@@ -59,8 +59,8 @@ TEST(FixedSizeVariable, Size)
 {
   // Verify the expected size is returned
   TestVariable variable;
-  EXPECT_EQ(2, variable.size());  // base class interface
-  EXPECT_EQ(2, TestVariable::SIZE);  // static member variable
+  EXPECT_EQ(2u, variable.size());  // base class interface
+  EXPECT_EQ(2ul, TestVariable::SIZE);  // static member variable
```

Comment:
weird. Why does that one have to be `ul`, but the others just `u`?

Reply:
I just moused over on `TestVariable::SIZE` and saw its type, and went with it. They really only have to be the same sign. I can change it if it bothers you. :)